src/ExplainAnalyzer.php

```
@@ -172,7 +181,8 @@ private function matchesPattern(array $explainResult, array $warnings, array $pa
     private function matchExplainPattern(array $explainResult, string $key, mixed $value): bool
     {
         if (isset($explainResult['query_block'])) {
-            if ($this->findInArray($explainResult['query_block'], $key, $value)) {
+            $walker = new ExplainWalker();
+            if ($walker->contains($explainResult['query_block'], $key, $value)) {
```

PHPStan (2.2.1)

[error] Parameter `#1` $node of method Koriym\SqlQuality\ExplainWalker::contains() expects array<string, mixed>, mixed given.


(argument.type)